src/App.js

```diff
@@ -8,7 +8,7 @@ class App extends Component {
       <div className="App">
         <section>
           <Calendar />
-          <a target="_blank" title="Contribute" href="https://github.com/itamark/the-daily-method"><img className="githubImage" src={Github}></img></a>
+          <a target="_blank" rel="noopener noreferrer" title="Contribute" href="https://github.com/itamark/the-daily-method"><img className="githubImage" src={Github}></img></a>
```

Comment:
Not sure if you want to enforce code conventions here, we use 120 max length per line.
For readability, I would break the img tag to a different line.
And tab instead of spaces, how I missed this debate :-)